llvm-spirv/lib/SPIRV/PreprocessMetadata.cpp

```diff
@@ -188,16 +188,17 @@ void PreprocessMetadataBase::visit(Module *M) {
     //         i32 Y, i32 Z}
     if (MDNode *MaxWorkgroupSizeINTEL =
             Kernel.getMetadata(kSPIR2MD::MaxWGSize)) {
-      assert(MaxWorkgroupSizeINTEL->getNumOperands() == 3 &&
-             "max_work_group_size does not have 3 operands.");
+      assert(MaxWorkgroupSizeINTEL->getNumOperands() >= 1 &&
```

Comment:
Our version of `llvm-spirv` is considered to be read-only, or at least we strive to have it as such. Unless there is a strong justification, any changes to the translator should first go into the upstream project and then be cherry-picked or brought by a regular pulldown

Reply:
Ah right, I wasn't aware - thanks. I've created https://github.com/KhronosGroup/SPIRV-LLVM-Translator/pull/2711

Reply:
This was merged (thanks) and now pulled down, so we can resolve this.